src/main/java/com/twilio/oai/TwilioGoGenerator.java

```diff
@@ -20,6 +28,18 @@ public void postProcessParameter(final CodegenParameter parameter) {
         parameter.required = parameter.isPathParam;
     }
 
+    @Override
+    public void processOpenAPI(final OpenAPI openAPI) {
+        super.processOpenAPI(openAPI);
+
+        // Group operations together by tag. This gives us one file/post-process per resource.
+        openAPI
+            .getPaths()
+            .forEach((name, path) -> path
+                .readOperations()
+                .forEach(operation -> operation.addTagsItem(PathUtils.cleanPath(name))));
```

Comment:
Why not just add tags in our open api specs?

Reply:
It would add extra maintenance/config for API owners when we migrate to api-defs to OpenAPI. Thought it was simple enough to do here.